docs/2023/cQube/cQubeChat/updates/2023-07-28.md

```diff
@@ -4,13 +4,20 @@ author: Suyash Gautam
 ---
 
 ## Milestones
-- [ ] Give the description about Milestone 1
-- [ ] Give the description about Milestone 2
-- [ ] Give the description about Milestone 3
-- [ ] Give the description about Milestone 4
 
-## Screenshots / Videos 
+- Minor bug fixes regarding prompts.
```

Comment:
If these are the milestones achieved, then mark them as checked checkboxes or else unchecked checkboxes, not in points like this.